src/arrays/programs_involving_multidimensional_data/hard/q5/ChartoInt.java

```diff
@@ -0,0 +1,34 @@
+package arrays.programs_involving_multidimensional_data.hard.q5;
+
+/*
+Write a Java program that initializes a 2 in 13 2D array with random integer between 97 to 122 (Inclusive of both ends),
+and print it. The program will then convert each one of those integer values into a character and display each row of
+the 2D array on a separate line.
+Note: the characters within a line do not need any spaces in between them.
+
+Sample Output:
+    105 112 110 121 109 112 115 117 99 111 108 113 116
+    118 105 100 121 122 97 112 115 105 122 110 120 109
+    ipnympsucolqt
+    vidyzapsiznxm
+ */
+
+public class ChartoInt {
```

Comment:
For questions like this, we want to make certain the student is actually using an array. To do this, it is preferable to use a method or to give large input sizes (1000+).

Reply:
For this one, I would use a method.